controllers/volumereplicationgroup_controller.go

```diff
@@ -85,6 +85,10 @@ func (r *VolumeReplicationGroupReconciler) SetupWithManager(
 			handler.EnqueueRequestsFromMapFunc(r.pvcMapFunc),
 			builder.WithPredicates(pvcPredicateFunc()),
 		).
+		Watches(&source.Kind{Type: &corev1.PersistentVolumeClaim{}},
```

Comment:
I did not know we could watch the same resource with different predicates and filters, good to know.